src/relay/transforms/compiler_function_utils.h

```diff
@@ -71,6 +71,7 @@ namespace transforms {
  */
 class GlobalSymbolCache {
  public:
+  virtual ~GlobalSymbolCache();
```

Comment:
nit: for simplicity, we have the implmentation inlined here
```suggestion
  virtual ~GlobalSymbolCache() = default;
```

Reply:
> nit: for simplicity, we have the implmentation inlined here

Thanks, I did consider that.  Normally I prefer to keep as many details in the .cc file as possible, for better modularity.